stable/mlflow/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: mlflow
+description: A Helm chart for MLflow
+type: application
+version: 1.0.0
+home: https://www.mlflow.org/
+appVersion: "latest"
```

Comment:
It's bad practice to use `latest` so how about use the latest proper tag, 1.9.1?

https://hub.docker.com/layers/larribas/mlflow/1.9.1/images/sha256-05ccb0b46bfb60ff0f88ed0bb0fa006a6e581e2d537f13ebebfeb98d3a13b90b?context=explore

Reply:
```suggestion
appVersion: 1.9.1
```